viadot/config.py

```diff
@@ -36,15 +36,15 @@ def from_yaml(cls, path: str, key: Optional[str] = None) -> Config:
         CONFIG = Config()
 
 
-def get_source_config(key):
-    source_configs = CONFIG.get("sources")
+def get_source_config(key, config):
+    source_configs = config.get("sources")
     if source_configs is not None:
         for source_config in source_configs:
             if key in source_config.keys():
                 return source_configs[source_configs.index(source_config)][key]
 
 
-def get_source_credentials(key):
-    config = get_source_config(key)
+def get_source_credentials(key, config):
```

Comment:
`config` should be optional as otherwise you would break all existing sources that use this function

Reply:
Done